src/core/net_db.cc

```diff
@@ -149,13 +152,16 @@ void NetDb::Run() {
         }
         last_save = ts;
       }
+      // publishes router info to a floodfill every 40 minutes
```

Comment:
>every 40 minutes

Giving static values within comments defeats the purpose of what we're trying to achieve: flexibility :smile: This also applies elsewhere.